server/app/repository/ApplicationEventRepository.java

```diff
@@ -62,4 +67,27 @@ public ImmutableList<ApplicationEventModel> getEventsOrderByCreateTimeDesc(Long
                 queryProfileLocationBuilder.create("getEventsOrderByCreateTimeDesc"))
             .findList());
   }
+
+  public void setNote(
```

Comment:
**suggestion, non-blocking**: I'd suggest naming this something like `insertNoteEvent()` or similar, so it's clear it's actually inserting a note into the db and not just setting a note on some object that will be inserted later.